### PR TITLE
How we use runbooks for Octopus.com

### PR DESCRIPTION
**Pull request by wordlee**
Based on Paul's internal document for getting octopus.com up and running in case of an outage.

### CODE REVIEW

**Comment by robpearson** [comment]
**Initial feedback** 

Suggestions: 
– Title tweak: How we keep http://Octopus.com running smootly with Opeartions Runbooks. One second thought, I like how you have Runbooks at the beginning. 
– Add a Why secton to link to our tweet where we were down for 3 hours. (probably after the http://octopus.com section). We could say how our website is like the cobbler's children's shoes (or a better reference). i.e. We never had major problems so kept it simple. Then we had a 3 hour outage and couldn't redcover quickly b/c of our depedency on our cloud provider. 
– I'll do a closer pass to SEO optmise it but incident management and incident recovery are two top operations/runbook keywords. 
– I think it's a good start but I'd include a lot more detail from Paul's blog post and tell a story with it. Highlighting runbooks is brilliant but I think we need to hook people with a story. I would also include Paul's OCTO acronym and more detail on how it all works. Paul's confluence page is quite thorough and he highlights some important points. Anyone can do it.

**Comment by wordlee** [comment]
Sorry, I didn't frame this PR very well. This post is intended as an example of how we're using runbooks. I wasn't going for the full story, though I agree that could also be a post.

File: blog/2020-03/using-runbooks-at-octopus/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: How we use Operations Runbooks for Octopus.com
+description:
```
Comment (wordlee) [inline blog/2020-03/using-runbooks-at-octopus/index.md]:
```suggestion
description: How we're using runbooks at Octopus to make anybody on the team can restore Octopus.com if it ever goes down
```

File: blog/2020-03/using-runbooks-at-octopus/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: How we use Operations Runbooks for Octopus.com
+description: How we're using runbooks at Octopus so that anybody on the team can restore Octopus.com if it ever goes down
+author: Octopus
+visibility: private
+published: 3020-01-01
+metaImage: to-be-added-by-marketing
+bannerImage: to-be-added-by-marketing
```
Comment (wordlee) [inline blog/2020-03/using-runbooks-at-octopus/index.md]:
```suggestion
metaImage: 
bannerImage: 
```

**Pull request merged**